components/logins/examples/sync_pass_sql.rs

```diff
@@ -212,7 +213,7 @@ fn show_all(engine: &PasswordEngine) -> Result<Vec<String>> {
                 timestamp_to_string(rec.time_last_used)
             }
         ]);
-        v.push(rec.id.clone());
+        v.push(rec.guid.as_str().to_owned());
```

Comment:
probably better to just store the guid in v, e.g. rec.guid.clone()

Reply:
Done!